### PR TITLE
Changed names of folders to match the package names

### PR DESCRIPTION
**Pull request by misaochan**
Fixes issue #14 . The package is now free.nrw.commons

### CODE REVIEW

File: commons/AndroidManifest.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="fr.free.nrw.commons"
+    package="free.nrw.commons"
```
Comment (nicolas-raoul) [inline commons/AndroidManifest.xml]: Ah it should be `fr.free.nrw.commons`

**Comment by nicolas-raoul** [comment]
Would you mind doing a git rebase to make this change a single commit? Thanks!

File: commons/AndroidManifest.xml (hunk 1 above)
Comment (misaochan) [inline commons/AndroidManifest.xml]: I tried that initially but had a problem with it as it requires the addition of a 'fr' package in the middle of the package tiers and moving the 'free' package into it via refactor. The refactor seemed to break the links to the layout files, for example `resDialogText = R.string.crash_dialog_text` produces the error 'cannot resolve symbol R'.

Comment (nicolas-raoul) [inline commons/AndroidManifest.xml]: R is a generated class. Delete it and regenerate it.
A way to do this is to run mvn clean and mvn install.
mvn is described in the README.

I am at the embassy to meet the French Prime Minister :-)

Nicolas

Comment (misaochan) [inline commons/AndroidManifest.xml]: Okay, I'll try that, thanks. Enjoy your meeting.  :)

Comment (misaochan) [inline commons/AndroidManifest.xml]: I have deleted and regenerated the R file several times, but it does not appear to work. I consistently get this error: `Error:android-apt-compiler: [commons] invalid resource directory name: C:\Users\misao\Documents\GitHub\apps-android-commons\commons\res java`

I have also invalidated the cache and restarted IntelliJ, and checked for any false android.R imports that might block regeneration.

Comment (nicolas-raoul) [inline commons/AndroidManifest.xml]: Hum... package names can be anything, there is no reason that one would work better than the other... have you run a full-text search for the older names to see if one subsists? A grep -R on Linux or grep -r on Mac... by the way what OS are you using?

Comment (misaochan) [inline commons/AndroidManifest.xml]: I think the reason free.nrw.commons works and fr.free.nrw.commons doesn't is that the former has 3 'tiers' in the folder hierarchy, similar to org.wikimedia.commons, and the latter has 4. In order to get the latter project structure to be valid in the IntelliJ IDEA, I had to create a new folder 'fr' and move the 'free' folder into it. Whereas with the former, I only had to rename the folders. I could not figure out any way to add one tier in the middle of the folder hierarchy without doing the move. That was why initially I was hoping that you would accept free.nrw.commons... :) But then I realized that the package name probably must stay the same if the app has already been published.

What older name do you mean... the package name? But we are only changing the folders, yes, so the package names in the code would not have changed from the initial clone? Unfortunately I develop on a Windows desktop.

Comment (nicolas-raoul) [inline commons/AndroidManifest.xml]: You are right that the package names in the code does not need to be changed.
I meant searching for "free.nrw" without the "fr.", or old occurrences of "org.wikimedia.commons"
When everything should work but does not, Stackoverflow is a good place to ask, people there usually know immediately what the problem is or can point you to possible solutions, I ask questions there almost everyday :-)

Comment (misaochan) [inline commons/AndroidManifest.xml]: Thanks, I'll ask on StackOverflow.

Do you mind if I create a branch from your last commit and start working on the 'remove usage settings' bug in the meantime? I only have two days at home before I start traveling again on Tuesday so I was hoping to make the most of them. :)

Comment (nicolas-raoul) [inline commons/AndroidManifest.xml]: Of course!
Working in branches is always a good idea.

**Pull request closed without merging**